src/test/java/org/dependencytrack/integrations/defectdojo/DefectDojoClientTest.java

```diff
@@ -88,31 +85,51 @@ public void testUploadFindingsPositiveCase() throws Exception {
         DefectDojoUploader uploader = new DefectDojoUploader();
         DefectDojoClient client = new DefectDojoClient(uploader, new URL("https://localhost/defectdojo"));
         client.uploadDependencyTrackFindings(token, engagementId, new NullInputStream(0));
+        testClient.verify(
+                request()
+                .withMethod("POST")
+                .withPath("/defectdojo/api/v2/import-scan/"),
+                VerificationTimes.exactly(1)
+        );
+        testClient.clear(
+                request()
+                    .withPath("/defectdojo/api/v2/import-scan/")
+        );
```

Comment:
Perhaps this clean up code should be in an `@After` method?

Reply:
I thought this - will change!